ports/espressif/boards/unexpectedmaker_feathers3/pins.c

```diff
@@ -103,18 +110,29 @@ STATIC const mp_rom_map_elem_t board_module_globals_table[] = {
     { MP_ROM_QSTR(MP_QSTR_VBUS), MP_ROM_PTR(&pin_GPIO34) },
     { MP_ROM_QSTR(MP_QSTR_VBUS_SENSE), MP_ROM_PTR(&pin_GPIO34) },
 
+    // Neopixel pins
     { MP_ROM_QSTR(MP_QSTR_NEOPIXEL_POWER), MP_ROM_PTR(&pin_GPIO39) },
     { MP_ROM_QSTR(MP_QSTR_NEOPIXEL), MP_ROM_PTR(&pin_GPIO40) },
 
-    { MP_ROM_QSTR(MP_QSTR_IO4), MP_ROM_PTR(&pin_GPIO4) }, // Ambient Light Sensor
-    { MP_ROM_QSTR(MP_QSTR_AMB), MP_ROM_PTR(&pin_GPIO4) }, // Ambient Light Sensor
+    // Ambient Light Sensor
+    { MP_ROM_QSTR(MP_QSTR_IO4), MP_ROM_PTR(&pin_GPIO4) },
+    { MP_ROM_QSTR(MP_QSTR_AMB), MP_ROM_PTR(&pin_GPIO4) },
 
-    { MP_ROM_QSTR(MP_QSTR_LDO2), MP_ROM_PTR(&pin_GPIO39) }, // Second LDO Enable control
-    { MP_ROM_QSTR(MP_QSTR_IO39), MP_ROM_PTR(&pin_GPIO39) }, // Second LDO Enable control
+    // Second LDO Enable control
+    { MP_ROM_QSTR(MP_QSTR_LDO2), MP_ROM_PTR(&pin_GPIO39) },
+    { MP_ROM_QSTR(MP_QSTR_IO39), MP_ROM_PTR(&pin_GPIO39) },
 
+    // I2C
     { MP_ROM_QSTR(MP_QSTR_I2C), MP_ROM_PTR(&board_i2c_obj) },
     { MP_ROM_QSTR(MP_QSTR_STEMMA_I2C), MP_ROM_PTR(&board_i2c_obj) },
+    { MP_ROM_QSTR(MP_QSTR_STEMMA_VERTICAL_I2C), MP_ROM_PTR(&board_stemma_i2c_obj) },
+    { MP_ROM_QSTR(MP_QSTR_STEMMA2_I2C), MP_ROM_PTR(&board_stemma_i2c_obj) },
+    { MP_ROM_QSTR(MP_QSTR_I2C2), MP_ROM_PTR(&board_stemma_i2c_obj) },
```

Comment:
I missed the `2` in `STEMMA2_I2C` the first time I read this. I'm wondering whether for consistency you might want to name these `STEMMA_I2C2` and `I2C2`, so that both use `I2C2`?

Reply:
Hey @dhalbert - I totally missed this, sorry.
I've destroyed by CP fork again) trying to rebase, and can't seem to find a way to fix it to get this change made.
If I re-fork, the PR will get killed right?
Can you rename it for me here in the PR?
Thanks :)

Reply:
@UnexpectedMaker  No problem. Let's try to start this over with just https://github.com/adafruit/circuitpython/pull/8174/commits/9642e387839c7fad564edc9909f22e8fcf2bec2e. The original commit can stand on its own, you don't need to rebase or merge.

I mostly avoid rebasing myself, because it's too easy to break things unless I am quite careful/

Getting back to the original point, do you agree with my idea to name them `STEMMA_I2C2` and I2C2`? If so, I think I can force-push the original commit and then make that chnage

Reply:
Or, try this yourself: `git checkout 9642e38; git push -f`.

That will get this PR back into shape, so you or I can then make the rename. After the PR is merged, you can delete the fork and re-fork.

In general, we very much advise making a branch for any PR. This PR is on `main`. You want your `main` to be the same as upstream. As a rule, don't make commits to `main` yourself: always use a branch. Then your main will track upstream.

Reply:
I tried that - It did get me back to the state I was in before I tried to add this new change, but as soon as I tried to rebase again so I could make the change and commit/push, I ended up in the same bad state again.

So instead, I just changed the file and committed it here in the web view, and once it's been merged, I'll stomp and start the fork again, and *try* to remember to work in branches!

Sorry for all of this trouble!